### DISCUSSION TITLE
Multiple PC's with Sunshine using same Public IP without VPN

### DISCUSSION THREAD

**Discussion by zjonesz** [LizardByte/community#493] [Sunshine]
Hi

Is there any solution for having multiple pc's running Sunshine using the same Public IP and still be able to connect to them?

Like NGINX but for Sunshine/Moonlight?

**Comment by buthig666** [comment]


**Reply by zjonesz** [reply]
Topic states "...without using VPN"
Tailacare is a VPN service, its built on top op WireGuard.

[a comment by an automated account is not shown]
